packages/taquito-rpc/test/taquito-rpc.spec.ts

```diff
@@ -50,6 +50,7 @@ import {
   delegatesKathmandunetSample,
   votingInfoKathmandunetSample,
   ticketUpdatesSample,
+  ticketBalances,
```

Comment:
might want to rename this to ticketBalancesResponse to make it a bit more descriptive